.github/workflows/style.yaml

```diff
@@ -2,12 +2,15 @@ name: Style
 on:
   push:
     tags:
-       - '*'
+    - '*'
     branches:
-      - master
-      - release-*
+    - master
```

Comment:
Are the whitespace/indentation changes opinionated from actionlint or elsewhere? And if so can this be enabled as a lintcheck so we all conform to the same standard?